hw/ip/i2c/doc/_index.md

```diff
@@ -67,8 +68,8 @@ This IP presents a simple register interface and state-machine to manage the cor
 ## Compatibility
 
 This IP block should be compatible with any target device covered by the [I2C specification](https://www.nxp.com/docs/en/user-guide/UM10204.pdf), operating at speeds up to 1 Mbaud.
-In order to support any target device, this IP issues addresses in 7-bit encoding whenever possible, as not all target devices may be able to support 10-bit encoding.
-(It remains the obligation of system designers to ensure that devices which cannot support 10-bit encoding remain in a 7-bit address space.)
+This IP in the host mode issues addresses in 7-bit encoding, and in the target mode, receives addresses in 7-bit encoding.
+(It remains the obligation of system designers to ensure that devices remain in a 7-bit address space.)
```

Comment:
I forgot if @mdhayter weighed in on the lack of 10-bit support here. Is that needed Mark?

Reply:
I have not seen 10-bit mode used. Most times you don't get more than a handful of devices (or the capacitance makes the bus too slow) and can avoid collisions with 7 bits.